src/components/TeamViewItem.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+export default class TeamViewItem extends React.Component{
+    constructor(props){
+        super(props);
+    }
+
+    render(){
+        // destructure object}
+        const {pokemon, onDeleteMe} = this.props
+        return(
+            <div className="team-view-item">
+                // IMAGE , NAME, TYPE
+                <p>
+                    <img className="">Pokemon Image Here</img> {pokemon.image}
```

Comment:
This should be: `<img alt="Pokemon Sprite" className="sprite" src={pokemon.sprites.front_default} />`

Though the alt and className could be anything along those lines.